code/headers/controller_interface.hpp

```diff
@@ -31,6 +31,12 @@ namespace r2d2::manual_control {
      */
     class controller_interface_c {
     public:
+        int controller_id;
```

Comment:
Is there a reason why a normal int was chosen instead of for instance a uint8_t?

Reply:
Changed